### PR TITLE
Fixes missing opacity in mobile menu

### LINKED ISSUE #135: Mobile menu lacks gradient background

### PR DESCRIPTION
**Pull request by alexandercyu**
Fixed opacity not showing in mobile menu. Fixes #135

### CODE REVIEW

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
Why put this in a media query at all? Right now its the exact same code in `@media (width < 576px)` and `@media (width >= 576px)` which is all widths

**Comment by alexandercyu** [comment]
> Why put this in a media query at all? Right now its the exact same code in `@media (width < 576px)` and `@media (width >= 576px)` which is all widths

Because only the one in `@media (width >992px) has a drop shadow.

**Comment by alexandercyu** [comment]
> Why put this in a media query at all? Right now its the exact same code in `@media (width < 576px)` and `@media (width >= 576px)` which is all widths

It's because the opacity would be visible on the desktop size horizontal menu.  The one at >992px disables the opacity and blur.

**Comment by mishaschwartz** [comment]
So then have a global setting and a setting that only specifies the change at >992px

**Comment by alexandercyu** [comment]
> So then have a global setting and a setting that only specifies the change at >992px

Moved this class out of the smaller media queries.

**Review by ShrutiK100 — APPROVED** [review]
Looks good!

File: static/css/marble.css [hunk 1]
```diff
@@ -1393,6 +1401,75 @@ h6{
         margin-right: 10px
     }
 
+    /*About*/
+    .margin-committee-title{
+        margin-bottom: 30px;
+    }
+}
+
+@media (width >= 576px) {
```
Comment (mishaschwartz) [inline static/css/marble.css:1410]: Doesn't this just add back the extra `@media (width >= 576px)` block that you condensed into one in the other PR?

Comment (alexandercyu) [reply static/css/marble.css:1410]: Pulled the changes for the >=576px media query into this branch.

**Review by mishaschwartz — APPROVED** [review]

**Pull request merged**